src/util-lua-base64lib.c

```diff
@@ -0,0 +1,104 @@
+/* Copyright (C) 2025 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+#include "suricata-common.h"
+#include "rust.h"
+#include "util-lua-base64lib.h"
+#include "util-validate.h"
+
+#include "lauxlib.h"
+
+static int LuaBase64EncodeStandard(lua_State *L)
+{
+    size_t input_len;
+    const char *input = luaL_checklstring(L, 1, &input_len);
+    size_t out_len = SCBase64EncodeBufferSize(input_len);
+    char output[out_len + 1];
```

Comment:
I do not like stack alloc... How big can it get ?

Reply:
I suppose quite large, but I suspect usually pretty small.  Does it make sense to change the allocation strategy base on size? If over 4k, malloc?

Reply:
It makes sense